jwst/transforms/models.py

```diff
@@ -50,10 +154,14 @@ class AngleFromGratingEquation(Model):
     inputs = ("lam", "alpha_in", "beta_in", "z")
     outputs = ("alpha_out", "beta_out", "zout")
 
-
     groove_density = Parameter()
     order = Parameter(default=-1)
 
+    def __init__(self, lam, alpha_in, beta_in, z, groove_density, order,
+                 name=None):
+        super(AngleFromGratingEquation, self).__init__(self,
+                        groove_density=groove_density, order=order, name=name)
+
```

Comment:
The inputs shoul dnot be in the `__init__` here. But actually remove the entire `__init__` , as it is not necessary.